### ISSUE TITLE
installation with conda env

### ISSUE THREAD

**Issue by asm582** [DIR-LAB/deep-batch-scheduler#5]
I have installed the below dependencies with conda:

# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
absl-py                   0.7.1                    pypi_0    pypi
astor                     0.8.0                    pypi_0    pypi
atari-py                  0.2.6                    pypi_0    pypi
atomicwrites              1.3.0                    pypi_0    pypi
attrs                     19.1.0                   pypi_0    pypi
backcall                  0.1.0                    pypi_0    pypi
binutils_impl_linux-64    2.31.1               h6176602_1  
binutils_linux-64         2.31.1               h6176602_9  
box2d-py                  2.3.8                    pypi_0    pypi
ca-certificates           2020.10.14                    0  
certifi                   2020.6.20          pyhd3eb1b0_3  
cloudpickle               0.5.2                    pypi_0    pypi
cycler                    0.10.0                   pypi_0    pypi
decorator                 4.4.0                    pypi_0    pypi
future                    0.17.1                   pypi_0    pypi
gast                      0.2.2                    pypi_0    pypi
gcc_impl_linux-64         7.3.0                habb00fd_1  
gcc_linux-64              7.3.0                h553295d_9  
google-pasta              0.1.7                    pypi_0    pypi
grpcio                    1.23.0                   pypi_0    pypi
gym                       0.12.5                    dev_0    <develop>
h5py                      2.9.0                    pypi_0    pypi
importlib-metadata        0.19                     pypi_0    pypi
ipython                   7.7.0                    pypi_0    pypi
ipython-genutils          0.2.0                    pypi_0    pypi
jedi                      0.15.1                   pypi_0    pypi
joblib                    0.13.2                   pypi_0    pypi
keras-applications        1.0.8                    pypi_0    pypi
keras-preprocessing       1.1.0                    pypi_0    pypi
kiwisolver                1.1.0                    pypi_0    pypi
ld_impl_linux-64          2.33.1               h53a641e_7  
libedit                   3.1.20191231         h14c3975_1  
libffi                    3.3                  he6710b0_2  
libgcc-ng                 9.1.0                hdf63c60_0  
libgfortran-ng            7.3.0                hdf63c60_0  
libstdcxx-ng              9.1.0                hdf63c60_0  
markdown                  3.1.1                    pypi_0    pypi
matplotlib                3.0.2                    pypi_0    pypi
more-itertools            7.2.0                    pypi_0    pypi
mpi                       1.0                     openmpi  
mpi4py                    3.0.2                    pypi_0    pypi
ncurses                   6.2                  he6710b0_1  
numpy                     1.17.0                   pypi_0    pypi
opencv-python             4.1.0.25                 pypi_0    pypi
openmpi                   4.0.2                hb1b8bf9_1  
openmpi-mpicc             4.0.2                h7b6447c_1  
openssl                   1.1.1h               h7b6447c_0  
packaging                 19.1                     pypi_0    pypi
pandas                    0.25.1                   pypi_0    pypi
parso                     0.5.1                    pypi_0    pypi
pexpect                   4.7.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pillow                    6.1.0                    pypi_0    pypi
pip                       20.2.4           py37h06a4308_0  
pluggy                    0.12.0                   pypi_0    pypi
prompt-toolkit            2.0.9                    pypi_0    pypi
protobuf                  3.9.1                    pypi_0    pypi
psutil                    5.6.3                    pypi_0    pypi
ptyprocess                0.6.0                    pypi_0    pypi
py                        1.8.0                    pypi_0    pypi
pyglet                    1.3.2                    pypi_0    pypi
pygments                  2.4.2                    pypi_0    pypi
pyparsing                 2.4.2                    pypi_0    pypi
pytest                    5.1.1                    pypi_0    pypi
python                    3.7.9                h7579374_0  
python-dateutil           2.8.0                    pypi_0    pypi
pytz                      2019.2                   pypi_0    pypi
readline                  8.0                  h7b6447c_0  
scipy                     1.3.1                    pypi_0    pypi
seaborn                   0.8.1                    pypi_0    pypi
setuptools                50.3.1           py37h06a4308_1  
six                       1.12.0                   pypi_0    pypi
spinup                    0.1.2                     dev_0    <develop>
sqlite                    3.33.0               h62c20be_0  
tensorboard               1.14.0                   pypi_0    pypi
tensorflow                1.14.0                   pypi_0    pypi
tensorflow-estimator      1.14.0                   pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
tk                        8.6.10               hbc83047_0  
torch                     1.4.0                    pypi_0    pypi
tqdm                      4.34.0                   pypi_0    pypi
traitlets                 4.3.2                    pypi_0    pypi
wcwidth                   0.1.7                    pypi_0    pypi
werkzeug                  0.15.5                   pypi_0    pypi
wheel                     0.35.1                     py_0  
wrapt                     1.11.2                   pypi_0    pypi
xz                        5.2.5                h7b6447c_0  
zipp                      0.5.2                    pypi_0    pypi
zlib                      1.2.11               h7b6447c_3

Hope this helps

**Comment by asm582** [comment]
closing the issue

**Comment by asm582** [comment]
I installed GCC as follows:
conda install -c omgarcia gcc-6
